src/mcwf.jl

```diff
@@ -322,14 +322,15 @@ Integrate a single Monte Carlo wave function trajectory.
 * `kwargs`: Further arguments are passed on to the ode solver.
 """
 function integrate_mcwf(dmcwf::T, jumpfun::J, tspan,
-                        psi0, seed, fout;
```

Comment:
I can't get the formatting to match here may be a tab vs spaces thing

Reply:
the diff tool gets confused occasionally -- this probably will be resolved when we squash